src/api/dataset.ts

```diff
@@ -23,8 +23,8 @@ export const datasetListQuery =
       growthConditions
       metadataJson
       status
-      fdrCounts {
-        level
+      fdrCounts(minFdr: 0, maxFdr: 60) {
```

Comment:
60 is a bit weird number here. I'd set maxFdr to 50 unless @andy-d-palmer has any other suggestions.